cli/args/flags.rs

```diff
@@ -1660,7 +1660,6 @@ fn add_dev_arg() -> Arg {
   Arg::new("dev")
     .long("dev")
     .short('D')
-    .help("Add as a dev dependency")
     .long_help("Add the package as a dev dependency. Note: This only applies when adding to a `package.json` file.")
```

Comment:
```suggestion
    .help("Add the package as a dev dependency. Note: This only applies when adding to a `package.json` file.")
```
we use `.help` everywhere, not `.long_help`. nowadays we use it for metadata for some internal handling on some specific flags. maybe we should add `long_help` to the banned types?

Reply:
Good catch, it was supposed to be `help`. Feel free to add it to banned types 👍